eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -540,12 +530,11 @@ govTransition = do
     foldlM' processProposal st $
       indexedGovProps (SSeq.fromStrict (OSet.toStrictSeq gsProposalProcedures))
 
-  -- Inversion of the keys in VotingProcedures, where we can find the voters for every
-  -- govActionId
-  let (unknownGovActionIds, knownVotes, replacedVotes) =
+  let knownVotes = [(voter, gas) | (voter, _vp, gas) <- knownVotesWithProcedures]
+      (unknownGovActionIds, !knownVotesWithProcedures, replacedVotes) =
```

Comment:
I wanted to point out that `knownVotesWithProcedures` is not quite accurate, but... `knownVotersWithGAS` ?? Probably that's worse :laughing:  It's just a local variable anyway, so good enough for me.